build.sbt

```
@@ -631,6 +635,32 @@ lazy val filtering = (project in file("modules/filtering"))
   .settings(name := "trace4cats-filtering", libraryDependencies ++= Dependencies.test.map(_ % Test))
   .dependsOn(model, kernel, `exporter-stream`)
 
+lazy val `dynamic-sampling` = (project in file("modules/dynamic-sampling"))
+  .settings(publishSettings)
+  .settings(name := "trace4cats-dynamic-sampling", libraryDependencies ++= Dependencies.test.map(_ % Test))
+  .dependsOn(model, kernel, `rate-sampling`)
```

```suggestion
  .dependsOn(model, kernel, `rate-sampling`, test % "test->compile")
```

Doesn't work for me without this.